bot/src/infrastructure/http/WandererMapClient.ts

```
@@ -0,0 +1,41 @@
+ * Renamed from MapClient to avoid confusion with JavaScript's Map class
+ */
+export class WandererMapClient {
+  // private readonly _client: UnifiedESIClient;
+
+  constructor(_baseUrl: string, _apiKey: string) { // _apiKey not used currently
+    // this._client = new UnifiedESIClient({
+    //   baseUrl,
+    //   userAgent: 'EVE-Chart-Bot/1.0',
+    //   timeout: 10000,
+    // });
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)